frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi'
+import {
+  Box,
+  chakra,
+  Collapse,
+  Flex,
+  Icon,
+  Stack,
+  Text,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { BxsBank } from '~assets/icons/BxsBank'
+import { BxsLockAlt } from '~assets/icons/BxsLockAlt'
+import Link from '~components/Link'
+
+import { GovtMastheadIcon } from './GovtMastheadIcon'
+import { GovtMastheadItem } from './GovtMastheadItem'
+
+export const GovtMasthead = (): JSX.Element => {
+  const { isOpen, onToggle } = useDisclosure()
+
+  return (
+    <Box>
+      <chakra.button
+        bg="neutral.200"
+        py={{ base: '0.5rem', md: '0.375rem' }}
+        px={{ base: '1.5rem', md: '1.75rem', lg: '2rem' }}
+        textStyle={{ base: 'legal', md: 'caption-2' }}
+        onClick={onToggle}
+        display="flex"
+        width="100%"
+      >
+        <GovtMastheadIcon
+          fontSize="1rem"
+          mr={{ base: '0.25rem', lg: '0.5rem' }}
+          my={{ base: '0px', md: '2px' }}
+        />
+        <Flex alignItems="center" flexWrap="wrap">
+          <Text my="2px">A Singapore government agency website.&nbsp;</Text>
+          <Link tabIndex={0}>
+            How to identify{' '}
+            <Icon
+              as={isOpen ? BiChevronUp : BiChevronDown}
+              fontSize={{ base: '1rem', md: '1.25rem' }}
+            />
+          </Link>
+        </Flex>
+      </chakra.button>
+
+      <Collapse in={isOpen} animateOpacity>
+        <Stack
+          direction={{ base: 'column', md: 'row' }}
+          spacing={{ base: '1rem', md: '4.5rem', lg: '9.5rem' }}
+          bg="neutral.200"
+          px="2rem"
+          py={{ base: '1.5rem', md: '2.25rem', lg: '2.75rem' }}
+          textStyle={{ base: 'caption-2', lg: 'body-1' }}
+        >
+          <GovtMastheadItem
+            icon={BxsBank}
+            header={<>Official website links end with .gov.sg</>}
```

Comment:
nit: if you change the type to `string`, you should be able to pass in a string directly without the weird fragment

Reply:
done~